app/windows/attitude-bar-view.js

```diff
@@ -0,0 +1,16 @@
+var AttitudeBarView = require('../../app/views/AttitudeBarView')(Marionette, Backbone);
```

Comment:
File name wrong for this file.  This one is a window, the other you got correct which are views. Ie. Should be attitude-bar-window.js